recipes/sdrf-pipelines/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - duckdb
     - rdflib
     - pyarrow
+    - setuptools
```

Comment:
Can you maybe add a comment here and also create an upstream issues. This should not be needed and fixed upstream IMHO.

Thanks @fabianegli

Reply:
Hmmmm... [I did fix it upstream](https://github.com/bigbio/sdrf-pipelines/commit/8269917f1bba6f777432e132452c8a302cefc705) and because it didn't propagate I thought I needed to do it here instead.

Re "Can you maybe add a comment here" Do you mean inside the yaml?

Reply:
PS `setuptools` is already listed in the requirements of the current PyPI package.